website/source/guides/external/index.html.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: "guides"
+page_title: "External Plugins"
+sidebar_current: "guides-external"
+description: |-
+  External plugins guides list
+---
+
+## External Plugins
+
+Starting with Nomad 0.9, task and device drivers are now pluggable. This gives users the flexibility to introduce their own drivers without having to recompile Nomad. See the navigation menu on the left or the list below for guides on how to some of the external drivers available for Nomad.
```

Comment:
How about "the flexibility to introduce their own drivers by providing a binary, rather than having to fork Nomad to make code changes to it".